games-simulation/simutrans/simutrans-nightly.recipe

```diff
@@ -9,18 +9,17 @@ changes which make your savegames unreadable by any official version, or \
 bugs that will destroy your game, or not even run. They are for bug testing \
 only."
 HOMEPAGE="http://www.simutrans.com"
-SOURCE_URI="svn://tron.yamagi.org/simutrans/simutrans/trunk \
-	--username=anon --password="
+SOURCE_URI="svn://servers.simutrans.org/simutrans/trunk"
 SOURCE_FILENAME="simutrans-$portVersion"
 SOURCE_DIR="trunk"
 COPYRIGHT="1997-2004 Hj. Malthaner
-	2005-2015 The Simutrans Team"
+	2005-2017 The Simutrans Team"
 LICENSE="Artistic"
-REVISION="2"
+REVISION="3"
 ARCHITECTURES="!x86_gcc2 ?x86 x86_64"
 SECONDARY_ARCHITECTURES="x86"
 PROVIDES="
-	simutrans$secondaryArchSuffix = $portVersion
+	simutrans$secondaryArchSuffix = $portVersion compat >= 120.2
```

Comment:
> (unfortunately, haikuporter sees "nightly" as a newer revision and would build it if you don't specify the version).

You can use 1.2.1~nightly for nightly builds before the 1.2.1 release. Using just "nightly" as a version should be avoided because there is no way to compare it to anything else.

I'm not sure the packsets need to depend on the game. You may want to install the packset for other reasons (extracting the data for display outside the game, using it with another game that would be compatible, etc). So, a strict dependency seems a bit too hard of a constraint. But we can express "softer" dependencies in our packages, and this may be a good place to use that.